ch05.adoc

```diff
@@ -192,6 +192,11 @@ A __grid mapping variable__ provides this information.
 If no __grid mapping variable__ is provided and the coordinate variables for a horizontal grid are not longitude and latitude, then it is required that the latitude and longitude coordinates are supplied via the `coordinates` attribute. 
 Such coordinates may be provided in addition to the provision of a __grid mapping variable__, but that is not required.
 
+When a data variable is representative of cells of non-zero size, and the coordinate variables are not longitude and latitude, __bounds__ variables should be provided for vertices of
+the cell boundaries in the eastings and northings of the grid (see  <<cell-boundaries>>). The __grid mapping variable__ provides then the information to convert bounds
+in latitude and longitude coordinates, and it is optional for the dataset to provide these.
```

Comment:
join this line to the previous one - following the "style", we have one sentence per line.